scaleapi/api.py

```diff
@@ -30,10 +30,20 @@ def __init__(self, api_key, user_agent_extension=None):
             "Content-Type": "application/json",
             "User-Agent": self._generate_useragent(user_agent_extension),
         }
+        self._headers_multipart_form_data = {
```

Comment:
> if you specify a files parameter (a dictionary), then requests will send a multipart/form-data POST instead of a application/x-www-form-urlencoded POST. 

ref: https://stackoverflow.com/questions/12385179/how-to-send-a-multipart-form-data-with-requests-in-python